src/main/flight/pid.c

```diff
@@ -842,6 +840,83 @@ void FAST_CODE applySmartFeedforward(int axis)
 }
 #endif // USE_SMART_FEEDFORWARD
 
+#if defined(USE_ABSOLUTE_CONTROL)
+static void applyAbsoluteControl(const int axis, const float gyroRate,
+    const float setpointLpf, const float setpointHpf,
+    float *currentPidSetpoint, float *itermErrorRate)
+{
+    if (acGain > 0) {
+        float acCorrection = 0;
+        float acErrorRate;
+
+        const float gmaxac = setpointLpf + 2 * setpointHpf;
+        const float gminac = setpointLpf - 2 * setpointHpf;
+        if (gyroRate >= gminac && gyroRate <= gmaxac) {
+            float acErrorRate1 = gmaxac - gyroRate;
```

Comment:
`const` for clarity and to help the compiler optimize `else` branch-only computation

Reply:
another thing from refactoring, but const can be added here of course